src/basic/src/ScalarSequence.C

```diff
@@ -1729,14 +1729,14 @@ ScalarSequence<T>::subBasicHistogram(
   for (unsigned int j = 0; j < dataSize; ++j) {
     double value = m_seq[j];
     if (value < minHorizontalValue) {
-      bins[0] += value;
+      bins[0]++;
```

Comment:
Oh wait, this change was showing up in an earlier commit too... You sure history is clean here? Or is GitHub not displaying this correctly?